documentation/high-avail/design/ha-design.md

```diff
@@ -0,0 +1,140 @@
+
+# Overview
+
+This document is a placeholder for the HA protocol proposal.  
+The purpose is to generate discussion that will lead to an actual design doc.
+
+
+# HA Requirements
+
+The High Availability document lists most of the requirements:
+https://github.com/Azure/DASH/blob/main/documentation/high-avail/design/high-availability-and-scale.md
+
+Microsoft has provided some additional requirements:
```

Comment:
Suggest these be moved to initial AH document per https://github.com/Azure/DASH/issues/57